__tests__/app.test.js

```diff
@@ -25,35 +25,36 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+});
 
-  describe("GET /api/topics", () => {
-    test("200: Responds with an array of all topics", () => {
-      return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then(({ body }) => {
-          const { topics } = body;
-          expect(topics.length).toBeGreaterThan(0);
-          topics.forEach((topic) => {
-            expect(topic).toEqual(
-              expect.objectContaining({
-                slug: expect.any(String),
-                description: expect.any(String),
-              })
-            );
-          });
-        });
-    });
-  });
-
-  test("404: Responds with an error when the endpoint does not exist", () => {
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of all topics", () => {
     return request(app)
-      .get("/api/non-existent-endpoint")
-      .expect(404)
+      .get("/api/topics")
+      .expect(200)
       .then(({ body }) => {
-        expect(body.msg).toBe("Endpoint not found");
+        const { topics } = body;
+        expect(topics.length).toBeGreaterThan(0);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
       });
   });
+});
+
+test("404: Responds with an error when the endpoint does not exist", () => {
+  return request(app)
+    .get("/api/non-existent-endpoint")
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Endpoint not found");
+    });
+});
 
 describe("GET /api/articles/:article_id", () => {
   test("200: Responds with an article object each with the correct properties", () => {
```

Comment:
Considering the request we are making here - do we want the article id to be ANY number? Or a specific one? because if article 5 is returned for example, this test would pass but we wouldn't expect it to.
Keeping the other keys flexible is fine though as we aren't concerned with what the specific title or author etc is - these can change in the test data and it's not a problem but if we request article 1 then we should always get back article number 1